apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -137,17 +137,16 @@ export function SideNav({}: Props) {
 
   return (
     <Navbar
-      p={24}
-      pt={16}
       sx={{
         position: 'sticky',
         top: 0,
         zIndex: 'auto',
         backgroundColor: 'transparent',
         borderRight: 'none',
-        paddingRight: 0,
         width: '300px',
-        height: 'max-content',
+        minHeight: '100vh',
```

Comment:
the navigation sidebar was causing layout problems on smaller screens like
![Screenshot 2023-08-11 at 01 02 39](https://github.com/novuhq/novu/assets/2607232/1e68fa69-9942-490b-98d6-88be6340d138)